### PR TITLE
[Fix] plugin file-structure doc rendering issue

### PR DESCRIPTION
**Pull request by Guaris**

### CODE REVIEW

**Comment by hutchic** [comment]
LGTM 
image

File: src/gateway/plugin-development/file-structure.md [hunk 1]
```diff
@@ -97,19 +97,19 @@ master each one of them.
...
-See [its source code] for more details.
-
-[api.lua]: {{page.book.chapters.admin-api}}
-[daos.lua]: {{page.book.chapters.custom-entities}}
-[handler.lua]: {{page.book.chapters.custom-logic}}
-[schema.lua]: {{page.book.chapters.plugin-configuration}}
-[migrations/*.lua]: {{page.book.chapters.custom-entities}}
-[Key-Auth plugin]: /hub/kong-inc/key-auth/
-[its source code]: https://github.com/Kong/kong/tree/master/kong/plugins/key-auth
+See the [Key-Auth source code] for more details.
```
Comment (cloudjumpercat) [inline src/gateway/plugin-development/file-structure.md:107]: The link here seems to be a 404 even though the link you have below works

File: src/gateway/plugin-development/file-structure.md [hunk 2]
```diff
@@ -97,19 +97,19 @@ master each one of them.
...
+
+[admin-api.lua]:(https://github.com/Kong/kong/blob/master/autodoc/admin-api/data/admin-api.lua)
+[daos.lua]:(https://github.com/Kong/kong/blob/master/kong/plugins/basic-auth/daos.lua)
+[handler.lua]:(https://github.com/Kong/kong/blob/master/kong/plugins/basic-auth/handler.lua)
+[schema.lua]:(https://github.com/Kong/kong/blob/master/kong/plugins/basic-auth/schema.lua)
+[migrations/*.lua]:(https://github.com/Kong/kong/tree/master/kong/plugins/basic-auth/migrations)
+[Key-Auth plugin]:(/hub/kong-inc/key-auth/)
+[Key-Auth source code]:(https://github.com/Kong/kong/tree/master/kong/plugins/key-auth)
```
Comment (cloudjumpercat) [inline src/gateway/plugin-development/file-structure.md:115]: Looks like all of these links (where they are in the published content) result in a 404.

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
There's some links that are 404 errors, but everything else LGTM!

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Pull request merged**